ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_namespace_functions.py

```diff
@@ -3060,6 +3060,37 @@ def test_jax_numpy_log10(
     )
 
 
+# degrees
+@handle_frontend_test(
+    fn_tree="jax.numpy.deg2rad",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
+    ),
+)
+def test_jax_numpy_deg2rad(
```

Comment:
`fn_tree` and the test name should refer to the function as jax.numpy.degrees, as that is the function name in the jax.numpy frontend.